panelapi/util.go

```diff
@@ -0,0 +1,29 @@
+package panelapi
+
+import (
+	"net/http"
+
+	"github.com/apex/log"
+)
+
+// Logs the request into the debug log with all of the important request bits.
+// The authorization key will be cleaned up before being output.
+//
+// TODO(schrej): Somehow only execute the logic when log level is debug.
```

Comment:
`c.Debug` in the config is the thing that sets debug on the logger.

Reply:
yeah, but panelapi shouldn't import the config

Reply:
Make it just have a debug flag passed in?